src/commands/bootstrap.js

```diff
@@ -1,5 +1,5 @@
 const eightball = require('./eightball');
-
+const help = require('./help')
```

Comment:
This is imported, but not added to the list of commands, so this command wont be available.

Reply:
Oops. Missed that line.

Reply:
Fixed